src/main/java/com/google/api/codegen/config/ApiConfig.java

```diff
@@ -70,18 +73,23 @@
   public abstract ImmutableMap<String, ResourceNameConfig> getResourceNameConfigs();
 
   /**
+   * Returns a map from field names to FieldConfigs for all fields that have a resource name type
+   * specified.
+   */
+  public abstract ImmutableMap<String, FieldConfig> getDefaultResourceNameFieldConfigMap();
```

Comment:
This is default in the sense that it is the field config that should apply outside of the context of a particular method.

I will rephrase/rename to make this clearer.